src/main/java/io/leonis/subra/math/spatial/GaussianPotentialField.java

```diff
@@ -35,12 +36,12 @@ public INDArray getForce(final INDArray positionVector) {
         Nd4j.zeros(positionVector.rows(), 1)
             .put(0, 0,
                 (sink ? -1 : 1) *
-                    (-2d * this.origin.add(positionVector).norm2Number().doubleValue()
-                        * StrictMath.pow(this.scale, 2)
+                    (-2d * this.height * this.origin.add(positionVector).norm2Number().doubleValue()
+                        * StrictMath.pow(this.height, 2)
```

Comment:
Why do you use `StrictMath` instead on `Math`; even though `Math` does not guarantee bit-for-bit equal results; it is likely to be more performant. as the ~dogs~ docs say:
>  Code generators are encouraged to use platform-specific native libraries or microprocessor instructions, where available, to provide higher-performance implementations of Math methods